chrome/renderer/BUILD.gn

```diff
@@ -172,6 +174,13 @@ static_library("renderer") {
     "//v8:v8",
   ]
 
+  if (enable_castanets) {
+    deps += [
+      "//third_party/wrt:wrt_lib",
+      "//third_party/wrt/ewk/efl_integration:chromium-ewk",
```

Comment:
How about merging 2 static libraries as one library?

Reply:
When I merged these 2 gn targets, I met gn cycle dependency problem.
So instead, I'm planning to use //third_party/wrt here,
and use below statement in third_party/wrt/BUILD.gn.

group("wrt") {
  public_deps = [
    ":wrt_lib",
    "ewk/efl_integration:chromium-ewk",
  ]
}